examples/baby-llama/baby-llama.cpp

```diff
@@ -1617,13 +1617,13 @@ int main(int argc, char ** argv) {
 
         float error_before_opt = ggml_get_f32_1d(e, 0);
 
-        struct ggml_opt_params opt_params_adam = ggml_opt_default_params(GGML_OPT_ADAM);
+        // struct ggml_opt_params opt_params_adam = ggml_opt_default_params(GGML_OPT_ADAM);
         struct ggml_opt_params opt_params_lbfgs = ggml_opt_default_params(GGML_OPT_LBFGS);
-        opt_params_adam.print_forward_graph = false;
-        opt_params_adam.print_backward_graph = false;
+        // opt_params_adam.print_forward_graph = false;
+        // opt_params_adam.print_backward_graph = false;
         opt_params_lbfgs.print_forward_graph = false;
         opt_params_lbfgs.print_backward_graph = false;
-        opt_params_adam.adam.n_iter = 16;
+        // opt_params_adam.adam.n_iter = 16;
```

Comment:
curious why we're commenting out instead of just removing?

Reply:
I commented it out to match the commented-out line below:
```
// ggml_opt(ctx0, opt_params_adam, e);
```
@xaedes Should opt_params_adam just be removed here? It's not currently used for anything.